amazoncorretto/issues.md

```diff
@@ -0,0 +1 @@
+https://github.com/corretto/corretto-8-docker/issues
```

Comment:
Since this is equivalent to the default (https://github.com/docker-library/docs/blob/0ae2dfcd0df23423f769d7bd9a27675df0ed190d/.template-helpers/issues.md), you should likely simply remove it, especially so that it gets linkified properly. :+1:

Reply:
All done!